src/main/kotlin/com/vaadin/gradle/VaadinUtils.kt

```diff
@@ -92,8 +96,11 @@ internal fun exec(logger: Logger, cwd: File, vararg args: String) {
     logger.info(result.outputUTF8())
 }
 
-internal fun JsonObject.writeToFile(file: File) {
-    file.writeText(JsonUtil.stringify(this, 2) + "\n")
+/**
```

Comment:
Let's try to keep all unrelated changes separately in the future, but this small doc chance is fine her.